ui/packages/catalog-ui-search/src/main/webapp/component/singletons/logout-actions.js

```diff
@@ -26,7 +26,7 @@ var LogoutAction = Backbone.AssociatedModel.extend({
 
 var LogoutActions = Backbone.Collection.extend({
     model: LogoutAction,
-    url: '/services/logout/actions'
+    url: '/search/catalog/internal/logout/actions'
```

Comment:
Pretty sure we want this to be `./internal/logout/actions` inline with upcoming changes in #3309 

If you get in first, it won't matter (it'll just break @brendan-hofmann 😄), but if he gets in first and the right circumstances happen (don't rerun CI after 3309 merges into master) then master will break